doc/guides/modules.rst

```diff
@@ -404,6 +404,35 @@ Module yaml file description
 A module can be described using a file named :file:`zephyr/module.yml`.
 The format of :file:`zephyr/module.yml` is described in the following:
 
+Module name
+===========
+
+Each Zephyr module is given a name by which it can be referred to in the build
+system.
+
+The name is specified in the :file:`zephyr/module.yml` file:
```

Comment:
How about "may be specified", since it is not mandatory?

Reply:
Better, thanks.